fabric-resource-loader-v1/src/main/resources/fabric.mod.json

```diff
@@ -28,6 +28,9 @@
     }
   ],
   "custom": {
-    "fabric-api:module-lifecycle": "stable"
+    "fabric-api:module-lifecycle": "stable",
+    "loom:injected_interfaces": {
+      "net/minecraft/class_6897": ["net/fabricmc/fabric/api/resource/v1/DataResourceStore"]
```

Comment:
Is this injected interface correct? class_6897 is an inner class of MinecraftServer, I see you are casting to this in the testmod this doesnt seem quite right. Let me know if you want me to take a look.